.travis/panic-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Check the unwrap and panic function presenting in src/ directory
+# of the repo. Testing suite is excluded. If such a function presents
+# return exit 1 for fail, otherwise return exit 0 for sucess.
+
+# Getting the banned list from banned functions
+banned_list=()
+banned_function="$1"
+while read -r line; do
+    banned_list=(${banned_list[@]} "$line")
+done < "$banned_function"
+
+output=""
+panic_line=0
+
+# Match panic call based on the given banned funciton list
+for i in src/*; do
+    for func in "${banned_list[@]}"; do
+	    tmp_line=$(sed '/#\[cfg(test)]/q' $i | grep -e "$func" | awk 'END {print NR}')
+        if [ "$tmp_line" -gt 0 ]; then
+            tmp_output=$(sed '/#\[cfg(test)]/q' $i | grep -n -e "$func")
+            output="${output}${tmp_output}\n"
+            panic_line=$(($panic_line+$tmp_line))
+        fi
+    done
+done
+
+# Output result
+if [ "$panic_line" -gt 0 ]
+then
```

Comment:
This goes on the previous line

Reply:
Just curious. Would it cause any difference or this is what people normally do?

Reply:
You did it yourself earlier in the file.  It's also pretty normal because you get a block structure similar to C (or Rust) - "then" corresponds to "{", and "fi" to "}".

Reply:
I see, thanks.